.github/workflows/release.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@main
```

Comment:
but this should not be `main` this should be `v1`, right? Otherwise you would be pulling the main to do a docker release for v1 images?

Reply:
Ahh this is self checkout, my bad..